### PR TITLE
Fix/ui flex

### PR DESCRIPTION
**Pull request by kaflan**

### CODE REVIEW

File: lib/api/open-test-in-editor.js [hunk 1]
```diff
@@ -7,6 +7,6 @@ module.exports = async (req, res) => {
     await openInEditor(file);
     res.json('OK');
   } catch (e) {
-    res.status(422).send({ message: e.stderr.toString().trim() });
+    res.status(422).send({ message: 'Editor can\'t be started' });
```
Comment (DavertMik) [inline lib/api/open-test-in-editor.js]: could you send, both error message & error description???

Comment (kaflan) [reply lib/api/open-test-in-editor.js]: ok

File: lib/api/open-test-in-editor.js [hunk 2]
```diff
@@ -7,6 +7,6 @@ module.exports = async (req, res) => {
     await openInEditor(file);
     res.json('OK');
   } catch (e) {
-    res.status(422).send({ message: e.stderr.toString().trim() });
+    res.status(422).send({ message: 'Editor can\'t be started', description: e.stderr.toString().trim() });
```
Comment (kaflan) [inline lib/api/open-test-in-editor.js]: here

**Comment by PeterNgTr** [comment]
@kaflan  Can we have a screenshot to see how it looks like?

**Pull request merged**